### DISCUSSION TITLE
All my pockhost instances are down

### DISCUSSION THREAD

**Discussion by Graciasjr** [pockethost/pockethost#346] [Q&A]
Hello pockhost, I just noticed that all my pockhost instances have been inactive for almost 1 week. But when I try to activate or access an instance, I get the http error `this requested resources wasn't found`, which means that all my sites which use my Pocketbase database hosted by you are also down due to lack of access to resources. I don't know what causes this problem and if the problem is general, but obviously not since I haven't seen any problem submissions along these lines.

**Comment by Jadi255** [comment]
I'm having the same issue, my instance is down

**Comment by benallfree** [comment]
Sorry everyone, there was an outage. Closing this issue in favor of #223
